### PR TITLE
Task #203654 and Task #204092 Reassign prerak

### PR DESCRIPTION
**Pull request by swapnilphalke03**
### I have ensured that following `Pull Request Checklist` is taken care of before sending this PR
* [ ] Code is formatted as per format decided
* [ ] Updated acceptance criteria in tracker
* [ ] Updated test cases in test-cases-tracker

https://tracker.tekdi.net/issues/203654
https://tracker.tekdi.net/issues/204092

### CODE REVIEW

File: apps/front-end/src/routes/admin.js [hunk 1]
```diff
@@ -72,10 +87,7 @@ export default [
     path: "/aadhaar-kyc/:id/:type",
     component: AdharKyc,
   },
-  {
-    path: "/admin/beneficiary/:id",
-    component: BeneficiaryAdminProfile,
```
Comment (manojLondhe) [inline apps/front-end/src/routes/admin.js:77]: BeneficiaryAdminProfile this is not needed?

File: apps/front-end/src/pages/admin/ReassignBeneficiaries/ReassignBeneficiariesList.js [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+  React.useEffect(async () => {
+    const learnerStatus =
+      await facilitatorRegistryService.learnerStatusDistribution(filter);
+    setPaginationTotalRows(learnerStatus?.data?.totalCount || 0);
+    setData(learnerStatus?.data?.data);
+    setLoading(false);
+  }, [filter]);
+
+  React.useEffect(async () => {
+    let name = "RAJASTHAN";
```
Comment (manojLondhe) [inline apps/front-end/src/pages/admin/ReassignBeneficiaries/ReassignBeneficiariesList.js:84]: in future we may need to show state dropwdown as well

**Comment by manojLondhe** [comment]
Also confirm if we are not writing duplicate code to show list of state, dist, blocks etc as we have already shown it at multiple places @swapnilphalke03

File: apps/front-end/src/routes/admin.js (hunk 1 above)
Comment (swapnilphalke03) [reply apps/front-end/src/routes/admin.js:77]: Added again. Resolved.

File: apps/front-end/src/pages/admin/ReassignBeneficiaries/ReassignBeneficiariesList.js (hunk 2 above)
Comment (swapnilphalke03) [reply apps/front-end/src/pages/admin/ReassignBeneficiaries/ReassignBeneficiariesList.js:84]: We are showing only district and block for filter, hence used state name directly and I believe for state we dont have a seprate API.

**Pull request merged**